pkgs/development/python-modules/manticore/default.nix

```
@@ -0,0 +1,111 @@
+{ lib
+, buildPythonPackage
+, capstone
+, crytic-compile
+, fetchFromGitHub
+, pkgs
```

```suggestion
, z3
, yices
```
This is more conventional than doing ``pkgs.z3``. This might also negate need for ``PATH=${pkgs.z3}/bin...``

---

`z3` will then be the Python library, I've tried that and it didn't work

---

In the ``python-packages.nix``, write
``manticore = callPackage ... { inherit (pkgs) z3; };